Ghidra/Features/Base/src/main/java/ghidra/app/util/opinion/OmfLoader.java

```diff
@@ -506,10 +581,15 @@ private void processExternalSymbols(OmfFileHeader header, Program program, TaskM
 				if (monitor.isCancelled()) {
 					break;
 				}
+				OmfSymbol sym = publicSymbols.get(symbol.getName());
+				if (sym != null) {
+					externsyms.add(sym);
+					continue;
```

Comment:
I'm confused by this 'continue'.  `symbol` is being used as if it's not `null` just below.

Reply:
The compare is on sym not symbol. This code checks if an external symbol has already been defined as a public symbol, and if that is the case, the already created symbol is reused, without creating a new one.
So sym could be renamed to something else to improve readability.

Reply:
Ah yes, sorry.  I misread that.